backend/src/v4/models/helper/model.js

```diff
@@ -658,7 +654,7 @@ function isSubModel(account, model) {
 
 		feds.forEach(modelSetting => {
 			promises.push(listSubModels(account, modelSetting._id).then(subModels => {
-				return subModels.find(subModel => subModel.model === model);
+				return subModels.find(subModel => subModel === model);
```

Comment:
This is a very dumb function to check if a model is a submodel....

All we need is a single query like in v5 https://github.com/3drepo/3drepo.io/blob/ISSUE_3845/backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/containers.js#L32